### ISSUE TITLE
Git pull not responding

### ISSUE THREAD

**Issue by jagafirst** [gitblit/gitblit#1155]
Gitblit is not responding when clone a repository until clone finished
I am using gitblit version v1.6.0

**Comment by gitblit** [comment]
Thanks for including the version, but this is not a lot of information to go on.  Perhaps you could elaborate with the following information:

- The variant I am using is: WAR, GO
- If WAR I am using this container: Tomcat, WildFly, Payara, other
- The OS I am using is: Windows, Linux, Mac
- The transport I am using to clone is: git://, http://, ssh://
- I decided that Gitblit *is not responding* because I observed....?

Updating to the current release wouldn't hurt.

**Comment by mmahann** [comment]
I'm seeing this as well. I am using 1.8.0 GO on linux, transport is ssh://. To test, I created a repo on the server and populated it with about 2GB data. When I git clone this repo on 3 machines concurrently, they are serviced in the order in which the git clone were started.

**Comment by gitblit** [comment]
@mmahann Try increasing the value of  `git.sshCommandStartThreads`.

**Comment by mmahann** [comment]
Default appears to be 2. Set to 4. Then started two concurrent git clone. First one starts clone, second one waits for first. If I stop first, second continues with clone. Set to 400, same result. 

I did the same experiment but used the http:// transport; both clones execute concurrently so this must be centered around ssh:// transport.

**Comment by gitblit** [comment]
Right, it should be that setting (requires a restart to apply).  Are you sure your config file is being properly loaded?

**Comment by mmahann** [comment]
Agreed that a restart is required. I've set it in both gitblit.properties and defaults.properties (just in case). I have rebooted and I know that my config files are being properly loaded because I assigned different http/https port numbers in gitblit.properties and I can reach the service on those ports.

**Comment by fzs** [comment]
Could be related or the same issue as in #1285.
